ruka/cnpg/cnpg.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+set -xe
+
+# Install cloudnativePG on cluster
+helm repo add cnpg https://cloudnative-pg.github.io/charts
+helm upgrade --install cnpg \
+  --namespace cnpg-system \
```

Comment:
"if set, upgrade process rolls back changes made in case of failed upgrade. The --wait flag will be set automatically if --atomic is used" -atomic was added in case of failed upgrade 👍